pelias-placeholder-service.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: pelias-placeholder
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: pelias-placeholder
+    spec:
+      initContainers:
+        - name: placeholder-download
+          image: busybox
+          command: ["sh", "-c",
+            "mkdir -p /data/placeholder/ &&\n
+             wget -O- http://pelias-data.s3.amazonaws.com/placeholder/graph.json.gz | gunzip > /data/placeholder/graph.json &\n
```

Comment:
Yes that was intentional, although I'm not sure if it really makes any noticeable difference. In any case this inline shell command format is pretty gross. Long term we definitely want a download script.